decidim-proposals/app/services/decidim/proposals/diff_renderer.rb

```diff
@@ -42,17 +18,6 @@ def attribute_types
           state: :string
         }
       end
-
-      def parse_changeset(attribute, values, type, diff)
-        diff.update(
-          attribute => {
-            type: type,
-            label: I18n.t(attribute, scope: "activemodel.attributes.collaborative_draft"),
-            old_value: values[0],
-            new_value: values[1]
-          }
-        )
-      end
     end
```

Comment:
The lines 36-59 of the file in `master` have been ommited from the backport as it is code that is not present in this branch:
https://github.com/decidim/decidim/blob/cb75e8aa52abb672b05681b0169ae4b1e2d31740/decidim-proposals/app/services/decidim/proposals/diff_renderer.rb#L36

Is this correct @oriolgual ?